instrumentation/mongodb-3.1/build.gradle

```diff
@@ -22,6 +22,10 @@ verifyInstrumentation {
     fails('org.mongodb:mongo-java-driver:[0.9.1,3.1.0-rc0)')
 }
 
+test {
+    systemProperty "jdk.util.zip.disableZip64ExtraFieldValidation", true
```

Comment:
Java 22 introduced more robust [zip file verification](https://bugs.openjdk.org/browse/JDK-8313765), which was failing on this test (and two others). It seems related to how some of the embedded database artifacts were compressed. This setting [disables](https://github.com/openjdk/jdk21u/pull/72/files#diff-b6128c3608aaf04e0802af64a2df5bb981e33bf873d41017df2c9ca4e187a545R1100) that extra validation.